CMakeLists.txt

```diff
@@ -180,14 +179,18 @@ include(CheckEndian)
 ###### Test Inputs ######
 
 if(GRAPH_LOCATION)
-  message(STATUS "Using graph location ${GRAPH_LOCATION}")
   set(BASEINPUT "${GRAPH_LOCATION}")
-elseif(EXISTS /net/ohm/export/iss/inputs)
+  set(BASEOUTPUT "${GRAPH_LOCATION}")
+  message(STATUS "Using graph input and output location ${GRAPH_LOCATION}")
+elseif(EXISTS /net/ohm/export/iss)
   set(BASEINPUT /net/ohm/export/iss/inputs)
-  MESSAGE(STATUS "Using graph location /net/ohm/export/iss/inputs")
+  MESSAGE(STATUS "Using graph input location /net/ohm/export/iss/inputs")
+  set(BASEOUTPUT /net/ohm/export/iss/dist-outputs)
+  MESSAGE(STATUS "Using graph output location /net/ohm/export/iss/dist-outputs")
 else()
   set(BASEINPUT "${PROJECT_BINARY_DIR}/inputs")
-  message(STATUS "Use 'make input' to download inputs in the build directory")
+  set(BASEOUTPUT "${PROJECT_BINARY_DIR}/inputs")
```

Comment:
should this be `${PROJECT_BINARY_DIR}/outputs`?

Reply:
No, it's intentional. For now, I'm keeping the inputs and outputs directory the same (except those hard-coded paths). I'll leave to Ian to change that if he changes "make input" to make them different folders.